vetiver/pin_read_write.py

```diff
@@ -112,3 +115,36 @@ def vetiver_pin_read(board, name: str, version: str = None) -> VetiverModel:
     v = VetiverModel.from_pin(board=board, name=name, version=version)
 
     return v
+
+
+def get_board_pkgs(board) -> List[str]:
+    """
+    Extract packages required for pin board authorization
+
+    Parameters
+    ----------
+    board:
+        A pin board, created by `pins.board_folder()` or another `board_` function.
+
+    Returns
+    --------
+    list[str]
+    """
```

Comment:
add example of a custom package or specific version

Reply:
Added a note on `prepare_docker` saying that more complex workflows with different requirements should use each individual `write_docker`, `load_pkgs`, and `write_app` function.